python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -463,7 +463,7 @@ def location(self):
                     LFNs.append(this_url)
             return LFNs
 
-    def accessURL(self, thisSE=''):
+    def accessURL(self, thisSE='', protocol='xroot'):
```

Comment:
A few comments. Could the default be a configuration parameter? In that case it can be different for different experiments. It would also be good if the help text gives a feeling of the possible options for this.